lib/check/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @type {script} query selector for files that are supposed to set themselves with the loaded flag
+ */
+const QUERY = 'script[src][onload*="this.loaded"]';
```

Comment:
Can be in `consts`.

Reply:
I think it's an integral part of the module's logic as opposed to the error message or code, which are arbitrary. I'd rather it being a visible part of it. Also, there is no reuse of this.